_episodes/08-ghpages-intro.md

```diff
@@ -16,6 +16,32 @@ GitHub Pages is a static site hosting service that hosts your website directly f
 
 [Jekyll](https://jekyllrb.com/) is used to generate your site from plain text to a static website, and [GitHub Pages](https://pages.github.com/) hosts the generated website. There are several [Jekyll Themes](http://jekyllthemes.org/) that you can choose from. For today, we will be working with the [Jekyll Now](http://www.jekyllnow.com/) theme designed by Barry Clark. It's a really quick and easy way to get up and running with a personal website and blog. 
 
-The first element of creating your website from a template is forking the template repository to your own GitHub repository, and giving it a specific name so that GitHub pages knows where to build the repository to. Go to the [Jekyll Now GitHub repository](https://github.com/barryclark/jekyll-now). Fork to your own repository by following these steps: 
+The first element of creating your website from a template is forking the template repository to your own GitHub repository, and giving it a specific name so that GitHub pages knows where to build the repository to. Go to the [Jekyll Now GitHub repository](https://github.com/barryclark/jekyll-now). Get started by following these steps:
 
-[!]../fig/fork.png
\ No newline at end of file
+**1. Fork the template GitHub repository to your own repository.** 
+
+![](../fig/fork.png)
+
+**2.  Go to the settings page of your forked repository.**
+
+![](../fig/settings.png)
+
+**3. Change the name of the repository as follows:**
+
+_yourusername.github.io_
+
+The name of this repository _must_ follow this format in order to render correctly. 
+
+**4. Voila! A functional static site at your specific address. Now to personalize it!**
+
+Before we start personalizing the Jekyll Now template, there are two ways you can go about it. You can either work through the command line using Jekyll, or work directly in GitHub Pages. We will show you both approaches, starting with the command line approach to practice the git tools we learned in the first half of the workshop. To do so, `cd` into a directory where you want to work on your website in. This can be your Desktop or any directory of your choosing. To clone the directory, click on the green clone/download button in your website repository: 
+
+![](../fig/clone.png)
+
+Once you are in the directory of your choosing, type in the following command: 
+
+```
+git clone https://github.com/lizilla1993/lizilla1993.github.io.git
+```
+
+Where the third argument is the copied address from your website repository. Now we can edit the Jekyll Now template from the command line and view our changes using Jekyll to build the static site at a temporary server before committing our changes to make the final product. 
```

Comment:
Where the third argument is the copied address from before or you replace USERNAME with your github username.

Now we can edit the Jekyll Now template from the command line and view our changes using Jekyll locally.  This allows us to build the static site at a temporary server on our computer thus allowing us to play around with the site before committing and pushing our changes to make the final version online.